### ISSUE TITLE
implement LM.kneighbors_graph

### ISSUE THREAD

**Issue by artemru** [tinyclues/labeledmatrix#19]
like in https://scikit-learn.org/0.15/modules/generated/sklearn.neighbors.kneighbors_graph.html#sklearn.neighbors.kneighbors_graph

* it should be easy for cosine or shadowsimilarity : 
-  X = X.normalize(axis=1, norm="l2") 
- mask = 1 - X.dot(X.transpose(), top=k+1)
- mask = mask.without_diagonal()
